.cargo/config.toml

```diff
@@ -1,6 +1,10 @@
 [target.x86_64-pc-windows-msvc]
 linker = "rust-lld"
 
+# Statically link the C runtime so the executable does not depend on the MSVC runtime DLLs.
+[target.'cfg(all(target_env = "msvc", target_os = "windows"))']
+rustflags = ["-C", "target-feature=+crt-static"]
```

Comment:
For what is that needed? :)

Reply:
As the comment says: to ensure that we link with the MSVC runtime statically and dont have a runtime dependency on the MSVC runtime.